idd/Energy+.idd.in

```diff
@@ -64671,6 +64671,208 @@ PlantEquipmentOperation:ComponentSetpoint,
        \key Cooling
        \key Dual
 
+PlantEquipmentOperation:ThermalEnergyStorage,
+       \memo Plant equipment operation scheme for simpler input to control thermal (ice)
+       \memo energy storage systems.  It replaces a host of setpoint managers with simple,
+       \memo single input values.  For more complex controls, use the ComponentSetpoint
+       \memo scheme.
+    \min-fields 7
+   A1, \field Name
+       \required-field
+       \reference ControlSchemeList
+   A2, \field On-Peak Schedule
+       \required-field
+       \type object-list
+       \object-list ScheduleNames
+   A3, \field Charging Availability Schedule
+       \required-field
+       \type object-list
+       \object-list ScheduleNames
+   N1, \field Non-Charging Chilled Water Temperature
+       \memo Single temperature for chiller outlet when not in cooling season
+       \memo or during on-peak cooling (discharge)
+       \type real
+       \units C
+   N2, \field Off-Peak Chilled Water Temperature
+       \memo Single temperature for chiller outlet when off-peak during cooling
+       \memo season (charging)
+   A4, \field Equipment 1 Object Type
```

Comment:
@RKStrand Also, You don't necessarily have to create a new combo object list name.  \object-list supports multiple lists in the same field, if that's easier.  e.g. 
```
   A12, \field Cooling Coil Name
        \required-field
        \type object-list
        \object-list CoolingCoilsDXSingleSpeed
        \object-list CoolingCoilsDXVariableSpeed
```